pages/pipelines/build_exports.md

```diff
@@ -88,26 +88,24 @@ Your Buildkite Organization ID (UUID) can be found on the settings page describe
 * Read and understand [Google Cloud Storage security best practices](https://cloud.google.com/security/best-practices) and [Best practices for Cloud Storage](https://cloud.google.com/storage/docs/).
 * Your bucket must have a policy allowing our Buildkite service-account access as described here and demonstrated in the example below¹.
```

Comment:
This should also be updated since the example has been moved.

```suggestion
* Your bucket must have a policy allowing our Buildkite service-account access as described here.
```